scripts/ci/Dockerfile

```diff
@@ -61,7 +61,7 @@ COPY --from=builder /build/${APP_HOME}/next-modules/ /webapp/next-modules/
 COPY --from=builder /build/${APP_HOME}/public /webapp/public
 COPY --from=builder /build/${APP_DIST_HOME}/.next /webapp/.next
 
-ENTRYPOINT [ "npx", "next", "start", "-p", "4200" ]
+ENTRYPOINT [ "node", "main.js" ]
```

Comment:
it would feel better to me if we would keep being explicit with the port in the Dockerfile

Reply:
IMO it's more appropriate to configure the port with environment variables. All of our server projects (Next, Nest and Express) support this using the `PORT` variable. At least they should be consistent.

We could add an `ENV PORT=4200` line to this container?

Reply:
Ah I didn't know that it could be configured via environment. Yes, please add it as a default env to the docker image 👍